### PR TITLE
New create goal form - personal goal switch

### PR DESCRIPTION
**Pull request by DenisVorop**
Create another issue to change tips to harmony view https://github.com/taskany-inc/issues/issues/2341
Change icon inside HelpButton, because Icon is not straight https://github.com/taskany-inc/bricks/issues/908

## PR includes

- [x] Feature

## Related issues

Resolve taskany-inc/issues#2095 
Resolve taskany-inc/issues#2361 

## QA Instructions, Screenshots, Recordings

Снимок экрана 2024-02-19 в 09 27 53

Снимок экрана 2024-02-19 в 09 31 43

https://github.com/taskany-inc/issues/assets/95316053/b38fc50d-b221-42ce-9f60-bdb31b8015b3

https://github.com/taskany-inc/issues/assets/95316053/86f44b26-8681-469a-bd25-b54ea6a946e7

### CODE REVIEW

**Comment by DenisVorop** [comment]
Снимок экрана 2024-02-19 в 13 03 33

**Comment by DenisVorop** [comment]
https://github.com/taskany-inc/issues/assets/95316053/45243265-d65f-40ff-ba4d-0964e7c4691f

**Review by awinogradov — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

File: src/components/GoalForm/GoalForm.tsx [hunk 1]
```diff
@@ -82,7 +97,9 @@ export const GoalForm: React.FC<GoalFormProps> = ({
     personal,
     ...attrs
 }) => {
-    const locale = useLocale();
+    const [personalView, setPersonalView] = useState<keyof typeof personalViewMap>(
```
[the remark this answers is not in the document]
Comment (DenisVorop) [reply src/components/GoalForm/GoalForm.tsx:100]: goals can be personal, personalView switches form to the personal view

[the remark this answers is not in the document]
Comment (asabotovich) [reply src/components/GoalForm/GoalForm.tsx:100]: it's some kind of `goalType/parentType`, I guess

**Pull request merged**